docs/learn/smart-contract-developers/getting-started.md

```diff
@@ -4,132 +4,342 @@ sidebar_position: 1
+
+  const abiEncoder = new ethers.AbiCoder();
+
+  // Encode constructor params
+  const encodedConstructorParams = abiEncoder.encode(
+    [
+      // your custom constructor types
+    ],
+    [
+      // your custom constructor parameters
+    ],
+  );
+
+  // Add the constructor params to the Custom Token bytecode
+  const CustomTokenBytecodeWithConstructor =
+    CustomTokenBytecode + encodedConstructorParams.slice(2);
+
+  // Get the address of the custom token contract that will be created
+  const CustomTokenAddress = await universalProfile
+    .connect(signer)
+    .getFunction('execute')
+    .staticCall(
+      1, // Operation type: CREATE
+      ethers.ZeroAddress,
+      0, // Value is empty
+      CustomTokenBytecodeWithConstructor,
+    );
+
+  // Deploy the contract by the Universal Profile
+  const tx = await universalProfile.connect(signer).getFunction('execute')(
+    1, // Operation type: CREATE
+    ethers.ZeroAddress,
+    0, // Value is empty
+    CustomTokenBytecodeWithConstructor,
+  );
+
+  await tx.wait();
+
+  console.log('Deployed contract address: ', CustomTokenAddress);
+}
+
+deployContract()
+  .then(() => process.exit(0))
+  .catch((error) => {
+    console.error(error);
+    process.exit(1);
+  });
+```
+
+  </TabItem>
+
+  <TabItem value="eoa" label="Deploy by EOA">
+
+```js title="deployMyCustomToken.ts"
+import { ethers } from 'hardhat';
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+async function deployContract() {
+  const [deployer] = await ethers.getSigners();
+  console.log('Deploying contracts with the account:', deployer.address);
+
+  const myCustomToken = await ethers.deployContract('MyCustomToken', [
+    // your custom constructor parameters
+  ]);
+
+  console.log('Deployed contract address:', await myCustomToken.getAddress());
+}
+
+deployContract()
+  .then(() => process.exit(0))
+  .catch((error) => {
+    console.error(error);
+    process.exit(1);
+  });
+```
+
+  </TabItem>
+
+</Tabs>
+
+You can then continue to run the deployment script:
+
+```bash
+npx hardhat --network luksoTestnet run scripts/deployMyCustomToken.ts
+```
+
+:::success Contract Examples
+
+You can find ready-to-go example contracts within the [`lukso-playground`](https://github.com/lukso-network/lukso-playground) repository.
+
+:::
+
+:::warning
+
+Please remember to always be careful when deploying smart contracts. Do not forget to process a security audit before deploying your smart contracts on production.
+
+:::
+
+## Verify smart contracts
+
+To verify a deployed contract, you can use the blockscout API properties set up within the `hardhat.config.ts`. You have to pass a file with all the constructor arguments that you've defined when deploying the smart contract, as well as the address of the deployed smart contract. The parameters and the compiled contract code are then used to verify the payload of the deployed contract on the address.
```

Comment:
maybe you can link:

`blockscout API` to https://explorer.execution.mainnet.lukso.network/api-docs

Reply:
That is actually a great find- thanks, will add it (but will put testnet one)